jelares/portfolio/src/main/webapp/javascript/script.js

```diff
@@ -0,0 +1,182 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Changes the text of an element to a randomly chosen string
+ * Side effect: switches the opacities of elementId + '1' and elementId + '2'
+ * Unless both have zero, at which point it initializes elementId + '1' with
+ * opacity 1 and elementId + '2' with opacity 0.
+ *
+ * @param {string[]} textChoices An array of strings from which the element text
+ *     is randomly chosen.
+ * @param {string} elementId The text is set on either elementId + '1' or
+ *     elementId + '2'. The text is added to the
+ * element which has zero opacity, then the element is given opacity 1  while
+ * the other is given opacity 0. This emulates a fading effect with CSS
+ * transitions.
+ */
+function addRandomText(textChoices, elementId) {
+  /* Pick a random text. */
+  const textChoice =
+      textChoices[Math.floor(Math.random() * textChoices.length)];
+
+  /* Add it to the page. */
+  const textContainer1 = document.getElementById(elementId + '1');
+  const textContainer2 = document.getElementById(elementId + '2');
+
+  /*
+      On page load both containers have 0 opacity and container1 will be chosen
+     by default.
+  */
+  if (textContainer1.style.opacity === '0' &&
+      textContainer2.style.opacity === '0') {
+    textContainer1.innerText = textChoice;
+    textContainer2.style.opacity = 0;
+    textContainer1.style.opacity = 1;
+
+    /*
+        Case when container 1 currently has text and opacity 1.
+    */
```

Comment:
Nit: consider making this comment single-line (as above) for conciseness:

```
/* Case when ... */
```

Reply:
Got it!